autoware_adapi_v1_msgs/vehicle/msg/VehiclePosition.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
+geometry_msgs/Pose pose
+autoware_adapi_v1_msgs/GeoPosition geo_position
```

Comment:
Autoware does not have a mechanism to provide a global pose, so I think it would be better to remove it or replace with MGRS code (empty string if not available).

Reply:
@isamu-takagi how about we change to relay the signal from gnss?
Which similar with https://github.com/tier4/tier4_ad_api_adaptor/tree/tier4/universe/awapi_awiv_adapter#awapivehiclegetstatus.

Reply:
@tkhmy Is it the data the application expects? In addition, some vehicles do not have GNSS and the accuracy is lower than pose.

Reply:
Because Autoware do not handle global location in some cases, we will need to discuss with the application team

Reply:
the geo position will be handle in array, if autoware cannot get the position, the api will pass an empty array
[d82c6e2](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/d82c6e22c4cff0fa15d4960ddae1425030e7f6c6)